pkgs/development/python-modules/schemainspect/default.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+
+  sqlalchemy,
+  setuptools,
+
+  # pytestCheckHook,
+  # pytest-cov,
+  # pytest-xdist,
+  # pytest-sugar,
+}:
+buildPythonPackage rec {
+  pname = "schemainspect";
+  version = "3.1.1663587362";
```

Likely needed when you fetch from github, along with the correct builder in `nativeBuildInputs`, such as `flit-core` or `poetry-core`

```suggestion
  version = "3.1.1663587362";
  format = "pyproject";
```